public/assets/js/login.js

```diff
@@ -0,0 +1,81 @@
+function toggleFunction() {
```

Comment:
What do you mean by toggleFunction()?
Is the function name clear enough to make you understand what the function does?

Reply:
Fixed it